### ISSUE TITLE
a11y: Color Contrast - Health Scale Component

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13806]
Insufficient color contrast for states of user interface controls

## Actual result

The mentioned progress bar on hover has an insufficient contrast ratio with background color making it difficult for people with low vision and color blindness to distinguish the user interface components in different states that have insufficient contrast. 

## Instances

"Foreground: #3D98D3
Background: #E2F0F8
Contrast ratio: 2.72:1 
Main content
- Health Progress Bar (on hover) "

## Expected result

The contrast ratio between the foreground color of the progress bar and its background color should be sufficient in the hover state.

**Comment by rak-phillip** [comment]
Issue 162 fails VPAT:

> "Actual Result:
> The mentioned progress bar on hover has an insufficient contrast ratio with background color making it difficult for people with low vision and color blindness to distinguish the user interface components in different states that have insufficient contrast. 
> 
> Instance:
> Foreground: #DCDEE7
> Background: #FFFFFF
> Contrast ratio: 1.3:1 
> Main content
> - Health Progress Bar( when no zero as value select for progress bar content)
> 
> Foreground: #6EA78C
> Background: #FFFFFF
> Contrast ratio: 2.8:1 
> Main content
> - Health Progress Bar( by defult)
> 
> Foreground: #6EA78C
> Background: #D7E1F5
> Contrast ratio: 2.1:1 
> Main content
> - Health Progress Bar (on hover) 
> 
>  
> Recommendation
> For the key parts of all the informative images, ensure that the contrast between foreground, background or adjacent colors is 3:1 to make the image's information accessible for low vision and color-blind users.
> 
> "

Issue 163 fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances:
> Foreground color: #BEC1D2
> Background color: #FFFFFF
> Contrast ratio: 1.78:1 
> Main content
> - Filter 
> 
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text.
> "
